index.bs

```diff
@@ -2053,24 +2053,20 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
                     1. Otherwise, [=list/Append=] |C| to |excludeCredentialDescriptorList|.
 
-                    <!-- Note: this next step is actually a step on the same level as the above, but bikeshed wanted it indented this much
-                    in order to render it as a numbered step. If outdented, it (today) is rendered either as a bullet in the midst
-                    of a numbered list or is mis-numbered -->
```

Comment:
Is it worth keeping this note since it is still technically indented one extra step relative to where it should be?

Reply:
Thanks for the fix!